classes/db/database.class.php

```diff
@@ -849,6 +850,8 @@ public function where($table, $whereArray = null) {
 							unset($or);
 						}
 					} else {
+						// Remove column label so that it can correctly check against table columns
+						$key = (is_string($key) && $label ? preg_replace("/^$label\./", '', $key, 1) : $key);
```

Comment:
@abrookbanks I think this same operation should also be done for the array values, but I wasn't sure:
```
$val= (is_string($val) && $label ? preg_replace("/^$label\./", '', $val, 1) : $val);
if (in_array($val, $allowed) && !is_numeric($val) || preg_match('/CONCAT/',$val)) {
```
If so, let me know and I'll update the PR.